pages/linux/pipes.sh.md

```diff
@@ -0,0 +1,24 @@
+# pipes.sh
+
+> A bash script which draws randomly pathed pipes over the terminal.
+> More information: <https://github.com/pipeseroni/pipes.sh>.
+
+- Display [v]ersion:
+
+`pipes.sh -v`
+
+- Change the pa[t]tern of the pipes:
+
+`pipes.sh -t {{0..9}}`
+
+- Change the [c]olor of the pipes:
+
+`pipes.sh -c {{0..7}}`
+
+- Change the [f]ramerate of the pipes:
+
+`pipes.sh -f {{20..100}}`
+
+- Disable [C]olors:
+
+`pipes.sh -C`
```

Comment:
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#help-and-version-commands
> We generally put, in this order, the help and version commands as the last two examples of the page
```suggestion
- Change the pa[t]tern of the pipes:

`pipes.sh -t {{0..9}}`

- Change the [c]olor of the pipes:

`pipes.sh -c {{0..7}}`

- Change the [f]ramerate of the pipes:

`pipes.sh -f {{20..100}}`

- Disable [C]olors:

`pipes.sh -C`

- Display [v]ersion:

`pipes.sh -v`
```